### ISSUE TITLE
Adjustment of hyper-parameters

### ISSUE THREAD

**Issue by Merrical** [hitcszx/ALFs#3]
Hi, 
Thanks for this great implementation. 
There are some hyper-parameters in the proposed ALFs. And the test accuracy corresponding to each hyper-parameter with different values is shown in the paper. However, there is no information on how to adjust the hyper-parameters in the training phase. So I wonder how you choose the value of these hyper-parameters. Is there a validation set with noisy labels? If yes, how to evaluate the performance on the noisy validation set? Looking forward to your reply.

Best Regards,
